src/CommandLineUtils/CommandOption.cs

```diff
@@ -101,6 +103,22 @@ internal CommandOption(CommandOptionType type)
         /// </summary>
         public List<string?> Values { get; } = new List<string?>();
 
+        /// <summary>
+        /// The default value of the option.
+        /// </summary>
+        public string? DefaultValue
+        {
+            get => _defaultValue;
+            set
+            {
+                _defaultValue = value;
+                if (value != null)
+                {
+                    Values.Add(value);
+                }
```

Comment:
Same here. I think the behavior users would expect is:


command line | `opt.DefaultValue` | `opt.Values`
--|--|--
`cmd`  | `"bar"`|  `{"bar"}` 
`cmd --option foo`  | `"bar"` |  `{"foo"}` 


But what is implemented here is:



command line | `opt.DefaultValue` | `opt.Values`
--|--|--
`cmd`  | `"bar"`|  `{"bar"}` 
`cmd --option foo`  | `"bar"` |  `{"bar", "foo"}`